docs/documentation/maya/skin.md

```diff
@@ -1,127 +1,189 @@
-# How To Use
+# AdnSkin
```

Comment:
Maybe we could expand a bit the page title, something along the lines of "Introduction to AdnSkin" or "AdnSkin documentation".